tensorboard/webapp/metrics/views/card_renderer/image_card_test.ts

```diff
@@ -1141,5 +1143,111 @@ describe('image card', () => {
         });
       });
     });
+
+    describe('selectedTime beyond range of data', () => {
+      it('clips the selectedTime to max step', () => {
```

Comment:
Also a little confused here: Are the first 2 of these 3 tests just testing existing functionality? I think only the 3rd one is a test for what is being added in this PR?